docs/_config.yml

```diff
@@ -18,10 +18,12 @@ collections:
   tutorials:
     output: true
     permalink: /tutorials/:slug
+  future:
+    output: true
+    permalink: /future/:slug
```

Comment:
Thoughts on grouping the Getting Started guide under /future/ as well?

![Screen Shot 2022-11-02 at 11 08 36 AM](https://user-images.githubusercontent.com/55667998/199568038-b1f49573-c213-4fb6-bfe0-56a6f156d10c.png)

Reply:
Ahh yess, I've debated this a few times, I opted for `/tutorial/` since eventually this will become a "normal" tutorial

I debated placing these directly in `/future/` or trying to do `/future/tutorial/`, let me know what you prefer

Reply:
Day 2 Sarah: not a huge deal either way actually, since it doesn't really impact the navigability. I will leave up to you!